lasagne/tests/test_nonlinearities.py

```diff
@@ -22,20 +22,33 @@ def sigmoid(self, x):
     def tanh(self, x):
         return np.tanh(x)
 
+    def scaled_tanh(self, x):
+        return np.tanh(x)
+
+    def scaled_tanh_p(self, x):
```

Comment:
The full thing should stand for `scaled_tanh_parametrized` as it is a test for the parametrized non standard `tanh`.